game/scripts/npc/items/item_blade_mail.txt

```diff
@@ -78,7 +78,7 @@
       "04"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "duration"                                        "3.0 3.5 4.0 4.5 5.0" // OAA
+        "duration"                                        "4.5"
```

Comment:
prais the lord!!!

Reply:
Blademail will maybe need more buffs. Because why would you ever upgrade this item into tier 2 Blade Mail if you are not Axe or Legion Commander. I always considered Blade Mail as a damage item for INT heroes, so maybe I should buff that aspect of the item.